src/tkn.ts

```diff
@@ -1305,19 +1305,20 @@ export class TknImpl implements Tkn {
     return data;
   }
 
-  async startPipeline(pipeline: StartObject): Promise<TektonNode[]> {
+  async startPipeline(pipeline: StartObject): Promise<string> {
     const result = await this.execute(Command.startPipeline(pipeline));
-    let data: TknTask[] = [];
-    try {
-      data = JSON.parse(result.stdout).items;
-      // eslint-disable-next-line no-empty
-    } catch (ignore) {
-
-    }
-    let pipelines: string[] = data.map((value) => value.metadata.name);
-    pipelines = [...new Set(pipelines)];
-    const treeState = pipelines.length > 0 ? TreeItemCollapsibleState.Expanded : TreeItemCollapsibleState.Collapsed;
-    return pipelines.map<TektonNode>((value) => new TektonNodeImpl(undefined, value, ContextType.PIPELINE, this, treeState)).sort(compareNodes);
+    // let data = [];
+    // try {
+    //   data = JSON.parse(result.stdout).items;
+    //   // eslint-disable-next-line no-empty
+    // } catch (ignore) {
+
+    // }
+    return result.stdout.match(/Pipelinerun|PipelineRun started:\s+([a-z0-9A-Z-]+)/)[1];
```

Comment:
I think we can create an issue for `tkn cli` for generating JSON output after starting the pipeline.

Reply:
In the doc, they have not provided `-o json` option https://github.com/tektoncd/cli/blob/master/docs/cmd/tkn_pipeline_start.md